### DISCUSSION TITLE
How can the quality of the svg file not be lost by zooming and it can be seen sharp

### DISCUSSION THREAD

**Discussion by saraOrkide** [pixijs/pixijs#9617] [General]
My photo svg with assets.load
I loaded it, I want the quality of the photo not to decrease with zoom in and zoom out and it is always shown sharp

**Comment by bigtimebuddy** [comment]
SVGs in Pixi are not infinitely scalable. GPU needs to rasterize the SVG. There are a few options: you can re-rasterize the SVG whenever you scale it, you can not use Pixi and use the DOM directly, or there are a few plugins that convert SVG into Graphics (https://www.npmjs.com/package/pixi-svg, https://www.npmjs.com/package/@pixi-essentials/svg) these are worth trying but the antialiasing isn't great and complex Graphics and lead to slowdown.

**Reply by saraOrkide** [reply]
Thank you very much, I will try
Of course, I used these two (https://www.npmjs.com/package/pixi-svg, https://www.npmjs.com/package/@pixi-essentials/svg), but many svg items I don't have it, I think I'll try this way of re-rastering and see how it goes
